### PR TITLE
Add namespace and delete function

### PR DESCRIPTION
**Pull request by gravidesaldrin**
Hi Sir. I add namespace for Admin, and index for user, and Delete function for Admin. Please review my PR. Thank You Sir.

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Admin::UsersController < AdminController
+  def destroy
+    user = User.find params[:id]
```
Comment (MuhammadTamzid) [inline app/controllers/admin/users_controller.rb]: use before action

File: app/controllers/admin_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class AdminController < ApplicationController
+  before_action :require_admin
+
+  private
+
```
Comment (MuhammadTamzid) [inline app/controllers/admin_controller.rb]: remove this line

**Comment by gravidesaldrin** [comment]
Ok Sir. Noted. Please for my update

File: app/assets/javascripts/application.js [hunk 3]
```diff
@@ -11,6 +11,12 @@
...
 //
 //= require jquery
+//= require bootstrap
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+
+$("[data-toggle=collapse]").click(function(){
+    // toggle icon
+    $(this).find("i").toggleClass("glyphicon-chevron-right glyphicon-chevron-down");
```
Comment (dieunb) [inline app/assets/javascripts/application.js]: 2 spaces

File: app/views/shared_pages/_sidebar.html.erb [hunk 4]
```diff
@@ -1,4 +1,22 @@
...
+      <strong><i class="glyphicon glyphicon-list">
+      </i> <%= t ".menu"%> </strong>
+    <% end %>
+    <hr>
+    <ul class="nav nav-stacked">
+      <ul class="nav nav-stacked collapse in" id="userMenu">
+        <li class="active">
+          <%= link_to users_path do %>
+            <i class="glyphicon glyphicon-user"></i>
+              <%= t ".all_users" %>
```
Comment (dieunb) [inline app/views/shared_pages/_sidebar.html.erb]: indent

File: app/views/users/_user.html.erb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<tr>
+  <td align="center">
+    <% if current_user.admin? && !current_user?(user) %>
+      <%= link_to [:admin, user], method: :delete, data: { confirm: "You sure?"} do %>
```
Comment (dieunb) [inline app/views/users/_user.html.erb]:
```
{confirm: "You sure?"}
```

i18n

File: app/views/users/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+        <h3 class="panel-title"><%= t ".title" %></h3>
+      </div>
+    </div>
+  </div>
+  <%= will_paginate %>
+  <div class="panel-body">
+    <table class="table table-striped table-bordered table-list">
+      <thead>
+        <tr>
+            <th><em class="glyphicon glyphicon-cog"></em></th>
```
Comment (dieunb) [inline app/views/users/index.html.erb]: indent

File: app/views/users/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+            <th><%= t ".name" %></th>
```
Comment (dieunb) [inline app/views/users/index.html.erb]: indent

File: app/views/users/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+            <th><%= t ".email" %></th>
```
Comment (dieunb) [inline app/views/users/index.html.erb]: indent

File: config/routes.rb [hunk 9]
```diff
@@ -7,5 +7,12 @@
   get "login" => "sessions#new"
   post "login" => "sessions#create"
   delete "logout" => "sessions#destroy"
-  resources :users, only: [:show, :new, :create, :edit, :update]
+
+  resources :users, only: [:index, :show, :new, :create, :edit, :update]
+  namespace :admin do
```
Comment (dieunb) [inline config/routes.rb]: root page of namespace

**Comment by gravidesaldrin** [comment]
Updated Sir. I add the root for Admin to the index of Category. But the Category will be present in the next PR. Thank You.

File: app/controllers/users_controller.rb [hunk 10]
```diff
@@ -1,7 +1,11 @@
 class UsersController < ApplicationController
-  before_action :logged_in_user, only: [:edit, :update]
+  before_action :logged_in_user, only: [:index, :show, :edit, :update]
   before_action :correct_user, only: [:edit, :update]
-  before_action :find_user, except: [:new, :create]
+  before_action :find_user, except: [:index, :new, :create]
+
+  def index
+    @users = User.paginate(page: params[:page])
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: remove `()`

**Pull request merged**